packages/neutrino-middleware-dev-server/index.js

```diff
@@ -0,0 +1,68 @@
+const { pathOr } = require('ramda');
+const opn = require('opn');
+
+module.exports = (neutrino, options = {}) => {
+  const { config } = neutrino;
+  const server = pathOr({}, ['options', 'server'], neutrino);
+  const protocol = process.env.HTTPS ? 'https' : 'http';
+  const publicHost = process.env.HOST;
+  const port = process.env.PORT || server.port || options.port || 5000;
+  const https = (protocol === 'https') || server.https || options.https;
+  let openInBrowser = false;
+  let serverPublic = false;
+  let host = 'localhost';
+
+  if (server.public !== undefined) {
+    serverPublic = Boolean(server.public);
+  } else if (options.public !== undefined) {
+    serverPublic = Boolean(options.public);
+  }
+
+  if (serverPublic) {
+    host = '0.0.0.0';
+  }
+
+  if (server.open !== undefined) {
+    openInBrowser = Boolean(server.open);
+  } else if (options.open !== undefined) {
+    openInBrowser = Boolean(options.open);
+  }
+
+  config
+    .devServer
+      .host(String(host))
+      .port(Number(port))
```

Comment:
Without environment variables in this middleware, this can just be `.port(port)`.